pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -130,23 +130,21 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxConfig(pod *v1.Pod, attemp
 
 // generatePodSandboxLinuxConfig generates LinuxPodSandboxConfig from v1.Pod.
 func (m *kubeGenericRuntimeManager) generatePodSandboxLinuxConfig(pod *v1.Pod, cgroupParent string) *runtimeapi.LinuxPodSandboxConfig {
-	if pod.Spec.SecurityContext == nil && cgroupParent == "" {
-		return nil
+	lc := &runtimeapi.LinuxPodSandboxConfig{
```

Comment:
The code is cleaner and it should not have any behavioural difference. I think it's worth doing it this way just for the sake of more readable code.

If you have a suggestion for something equally readable that retains the nil, happy to hear